assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -81,10 +91,41 @@ const EditorWizardModal = () => {
 		} );
 	};
 
+	// Choose steps by post type.
+	const stepsByPostType = {
+		course: [ CourseDetailsStep, UpgradeStep, CoursePatternsStep ],
+		lesson: [ LessonDetailsStep, LessonPatternsStep ],
+	};
+	const postType = select( 'core/editor' )?.getCurrentPostType();
```

Comment:
I think since we're in a React scope, it would be interesting to use the `useSelect` here.

```js
const { postType } = useSelect( select => ( {
    postType: select( editorStore )?.getCurrentPostType(),
} );
```

Notice that it's also interesting to use the `editorStore` here since it's the new way to do it.

Reply:
Changed here aff5ae0e0

Thanks!